sys-kernel/dracut/dracut-9999.ebuild

```diff
@@ -167,6 +170,11 @@ pkg_postinst() {
 		"Install ssh and scp along with config files and specified keys" \
 		virtual/openssh
 	optfeature "Enable logging with rsyslog" app-admin/rsyslog
+	optfeature "Support Squashfs" sys-fs/squashfs-tools
+	optfeature "Support TPM 2.0 TSS" app-crypt/tpm2-tools
+	optfeature "Support Bluetooth (experimental)" net-wireless/bluez
+	optfeature "Support BIOS-given device names" sys-apps/biosdevname
+	optfeature "Support NVMe" sys-apps/nvme-cli
```

Comment:
It this is for the 95nvmf module? If so, the description should mention that this is only needed for remote NVMe devices. It's not needed for NVMe storage attached directly to the PCIe bus.

Also, according to module-setup.sh, the "jq" command from app-misc/jq is also required.

Reply:
> Once upstream releases v60 and Gentoo upgrades, only than we should make this change I think.

Let's add it in the 9999 ebuild; I use that as the basis for any version bumps.